packages/web/public/static/locales/de/common.json

```diff
@@ -47,6 +47,8 @@
     "bodyOne": "Brauchst du Hilfe? Wir sind für dich da!",
     "bodyTwo": "Bitte sende eine E-Mail an ",
     "bodyThree": " mit einer detaillierten Beschreibung deines Problems und wir werden uns so schnell wie möglich bei dir melden!",
+    "bodyFour": "Have you seen concerning behavior on the platform? Please refer to our Terms of Service and always feel free to report anything you feel may be misaligned with our guidelines:",
+    "termsOfService": "Terms of Service",
```

Comment:
Hi @LindasLinguas ! Would you be able to translate these whenever you have the time? I think for now we don't need to worry about translating the actual Terms of Service document because it could be a lot of work and will likely evolve quite a lot as we learn more. I'm thinking perhaps once we have a very stable version of the document that has been checked by professionals, then we could go about translating it into our UI languages.

Reply:
"termsOfService": "Allgemeine Geschäftsbedingungen",

Reply:
German is soooo wordy :o